src/components/offer-list/offer-list.tsx

```diff
@@ -1,22 +1,22 @@
-import SortBlock from '../sort-block/sort-block';
-import { TOfferList } from '../../types';
+import { TCardSizes, TOfferList } from '../../types';
 import OfferCard from '../offer-card/offer-card';
 
 type TOfferListProps = {
   offerList: TOfferList;
+  listClassName: string;
+  cardSize: TCardSizes;
+  handleMouseEnter?: (id: string) => void;
+  handleMouseLeave?: () => void;
```

Comment:
Такие функции можно типизировать вот так:
```suggestion
  handleMouseLeave?: VoidFunction;
```